services/horizon/internal/db2/history/liquidity_pools.go

```diff
@@ -150,8 +155,30 @@ func (q *Q) FindLiquidityPoolByID(ctx context.Context, liquidityPoolID string) (
 	return lp, err
 }
 
-// GetLiquidityPools finds all liquidity pools where accountID is one of the claimants
+// findLiquidityPoolsByAccountId finds all liquidity pools that accountID is a participant of
+func (q *Q) findLiquidityPoolsByAccountId(ctx context.Context, accountID string) ([]LiquidityPool, error) {
+	var results []LiquidityPool
+	sql := selectLiquidityPoolsJoinedTrustlines.Where("lp.deleted = ?", false)
+	sql = sql.Where("trust_lines.account_id = ?", accountID)
+	if err := q.Select(ctx, &results, sql); err != nil {
+		return nil, errors.Wrap(err, "could not run select join query")
+	}
+
+	return results, nil
+}
+
+// GetLiquidityPools finds all liquidity pools where accountID owns assets
 func (q *Q) GetLiquidityPools(ctx context.Context, query LiquidityPoolsQuery) ([]LiquidityPool, error) {
+	log.Infof("%+v", query)
+	if len(query.AccountID) > 0 && len(query.Assets) > 0 {
+		return nil, fmt.Errorf("only one of `account id` or `assets` can be specified in a liquidity pool request")
```

Comment:
nit: wrong param names here. We can also stay consistent with other error message (see e.g. `TradesQuery.Validate()`):
> `errors.New("this endpoint does not support filtering by both asset pair and offer id ")`

```suggestion
		return nil, fmt.Errorf("this endpoint does not support filtering by both account ID and reserve assets.")
```

Reply:
Updated the error message here.